eShopModernizedWebFormsSolution/src/eShopModernizedWebForms/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/aspnet:4.7.1
+FROM microsoft/aspnet:4.7.2
```

Comment:
Can we pull images from mcr instead of Docker Hub?

Reply:
The syntax is like this, right?
`FROM mcr.microsoft.com/dotnet/core/aspnet:2.2`

Specifically for this I would want:
`FROM mcr.microsoft.com/aspnet:4.7.2`

Correct?

Reply:
Try this one `mcr.microsoft.com/dotnet/framework/aspnet:4.7.2` ?

You can see the TagList here: https://hub.docker.com/_/microsoft-dotnet-framework-aspnet/